lib/components/ContactList.js

```diff
@@ -3,11 +3,18 @@ import Link from './Link';
 
 
 const ContactList = ({ contacts }) => {
+
   let contactName = contacts.map((obj, i) => {
+    handleClick() {
```

Comment:
this should be: `const handleClick = () => {  ...  }`

Since you are inside an iterator, you have to assign the function to a variable. `functionName () { ... }` is shorthand ES6!